### DISCUSSION TITLE
internal.cmake  is the correct cmake file for CXXFLAGS  amd CFLAGS settings?

### DISCUSSION THREAD

**Discussion by osevan** [dragonflydb/dragonfly#97] [Q&A] [answered]
[helio](https://github.com/romange/helio)/[cmake](https://github.com/romange/helio/tree/master/cmake)/internal.cmake

Is this file correct, when i set my flags for dragonflydb or another file is needed?

**Comment by romange** [comment, marked as answer]
You can change there, yes...
